### PR TITLE
Test: add test + fix eslint for OnboardingUsers component

### PR DESCRIPTION
**Pull request by khendrikse**
## What type of PR is this? (check all applicable)

- [x] Refactor
- [ ] Feature
- [ ] Bug Fix
- [ ] Documentation Update

## Description
I saw there were a lot of eslint errors in the component so I tried to fix them as much as I could. I also wrote two extra tests and refactored the test a little for efficiency.

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [x] no documentation needed

## [optional] What gif best describes this PR or how it makes you feel?

doing stuff

### CODE REVIEW

**Comment by khendrikse** [comment]
I am wondering though if requiring the props is a good idea.

**Comment by rhymes** [comment]
Hi @khendrikse, thanks for the PR! 

I think you worked on the same set of files that this PR https://github.com/thepracticaldev/dev.to/pull/4390 worked on!

We have an index of who's working on what here: https://github.com/thepracticaldev/dev.to/issues/2470#issuecomment-537845194

I'l take a look at your PR nonetheless!

File: app/javascript/src/components/OnboardingUsers.jsx [hunk 1]
```diff
@@ -76,6 +72,8 @@ class OnboardingUsers extends Component {
 }
 
 OnboardingUsers.propTypes = {
+  users: PropTypes.arrayOf(PropTypes.object).isRequired,
+  checkedUsers: PropTypes.arrayOf(PropTypes.object).isRequired,
```
Comment (rhymes) [inline app/javascript/src/components/OnboardingUsers.jsx]: this should use the right type, see the other PR https://github.com/thepracticaldev/dev.to/pull/4261/files#diff-fb1929368bb321a5d6266d1a0db8c9e5R80

**Comment by khendrikse** [comment]
Ah I see that the onboarding files have been removed :) I'll close the PR!

**Pull request closed without merging**